petfinder-data/data_getter.py

```diff
@@ -69,7 +68,7 @@ def get_organizations() -> pd.DataFrame:
     headers = {"Authorization": f"Bearer {token}"}
 
     params = {
-        "location": "Chicago, IL",
+        "location": City_State,
```

Comment:
Since this is now a parameter, maybe we should make it a parameter in the `get_organizations` function definition?

Reply:
Done